producer-js/README.md

```diff
@@ -0,0 +1,235 @@
+## KVS Browser-based Ingestion ##
+
+
+There has been customer asks for sending video streams from browser to KVS. Browser environment is attractive to some customers because of its ubiquity and ease of deployment. The current solution that we recommend to customers is sending MKV files to S3 and using a lambda to do putMedia there. However, orchestrating this solution is not simple for customers as they need to do some heavy processing to get the audio and video data in browser, package it into MKV, and write lambdas for uploading to S3 and KVS. This solution is also slow - writing in S3 and having to use Java Producer SDK in lambda being long poles. 
+
+The aim of this project is to send audio and video stream from camera and microphone from your browser into a KVS stream without a proxy server or lambda. While doing so, ensuring that the video is in MKV format, uses H.264 codec for video and AAC for audio. 
+
+Currently available via Chrome, there four important tasks that this repository helps perform
+1. Obtaining the video from the webcam/ uploading video file
+2. Converting it to containers and codecs expected by the playback
+3. Sending to the putMedia API via POST calls
+4. Viewing the stream on the browser
+
+
+#### Setup as a Web-Application ####
+1. Clone the repository as
+```
+    git clone https://github.com/aws-samples/amazon-kinesis-video-streams-demos.git
+```
+
+2. Go to `browser-based-ingestion` folder. Add the `lib` and `src` folders from that folder to your project to use this repository. Include the code from `lib` into your scripts in your HTML page with the following 
+```
+    <script src="../lib/aws-sdk-2.714.2.min.js"></script>
```

Comment:
Since you've bundled all the dependencies into `main.js`, there's no need to include these dependencies anymore, otherwise you'll double include them.

Reply:
I have removed this, DS_Store, removed web-applications.